content/en/docs/trustee/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Trustee
+description: Trusted Components for Attestation and Secret Management
+weight: 50
+categories:
+- docs
+tags:
+- docs
+- trustee
+---
+
+Trustee contains tools and components for attesting confidential guests and providing secrets to them. Collectively, these components are known as Trustee. Trustee typically operates on behalf of the guest owner and interact remotely with [guest components](../guest-components/).
```

Comment:
it might be a good idea to align it with the personas in https://confidentialcontainers.org/docs/overview/trust-model/trust_model_personas/ "guest owner" is not a category in that model. it's probably "data owner"

Reply:
In the diagram https://confidentialcontainers.org/docs/overview/trust-model/trust_model_personas/, probably "workload provider" and "data owner". This is a little subtle because the attested thing is the whole guest (software stack, VM, ...), thus the one been on behalf of should aim to do attestation. In simple scenario, the workload provider deploys the guest and this is the only persona.

In a more complex, workload provider deploys the workload and do attestation. Also data owner does attestation to decide whether data could be accessed.